ansible/roles/stack-sunbird/templates/inbound.env

```diff
@@ -49,5 +49,5 @@ ENV={{env}}
 #Redis
 REDIS_HOST={{sunbird_redis_host}}
 REDIS_PASS=""
-REDIS_PORT=6379
+REDIS_PORT={{sunbird_redis_port | default(6379)}}
 REDIS_DB_INDEX={{redis_db_index_uci | default('6')}}
```

Comment:
Please change the default index to 7

Reply:
is there a variable available for redis password like "sunbird_redis_pass", or should I just add a new one here. 
Something like this should work right?
REDIS_PASS={{sunbird_redis_pass | default('')}}